insights/core/__init__.py

```diff
@@ -1007,12 +1007,12 @@ def token_scan(cls, result_key, token, check=all, reverse=False):
             reverse(bool): scan start from the head when ``False`` by default, otherwise start from the tail
 
         Returns:
-            (list): list of dictionaries corresponding to the parsed lines contain the `token`.
+            (bool): the property will contain True if a line contained (any
```

Comment:
@PaulWay I agree  that the `reverse` is not useful in `token_scan()` but it returns the `list` of `dict` as mentioned.

Reply:
Um, the code is: `return any(...)` - which returns a boolean.  If you're expecting a list of dict, you should be using `keep_scan` or `last_scan`.  The purpose of `token_scan` is to say "did I see this token anywhere in the output", but without returning any more information than that.